app/scripts/lib/multichain-api/scope/validation.ts

```diff
@@ -120,3 +128,30 @@ export const validateScopes = (
     validOptionalScopes,
   };
 };
+
+export const validateScopedPropertyEip3085 = (
```

Comment:
this needs a spec

Reply:
https://github.com/MetaMask/metamask-extension/pull/25873/commits/0aa8f29d0df4f49f8ceb17bf0bd9aa854cae86ec